api.raml

```diff
@@ -0,0 +1,418 @@
+              r: '7542baa842d9d1ed7a49cc8b93ac7a94a84802bb47f778e1bebb302cc73f9553',
+              s: '39137b296c3143b8d2418bb72fd76dc5f7ecc717b88ac3ef1b823b14d81cc856',
+              v: 27
+            }
+          }
+      responses:
+        202:
+          body:
+            example: |
+              {
+                "hash": "0x4e3fe115182219ac5d787bcb8d616bd29b73e19e6afc15def1ac999383d3642c",
+              }
+    put:
+      description: Used for send cancel order transaction
+      body:
+        example: |
+            {
+              tx: {
+                nonce: 18,
+                gasPrice: 1000000000,
+                gasLimit: 250000
+              },
+              signature: {
+                r: '7542baa842d9d1ed7a49cc8b93ac7a94a84802bb47f778e1bebb302cc73f9553',
+                s: '39137b296c3143b8d2418bb72fd76dc5f7ecc717b88ac3ef1b823b14d81cc856',
+                v: 27
+              }
+            }
+      responses:
+        201:
+          body:
+            example: |
+              {
+                "hash": "0x4e3fe115182219ac5d787bcb8d616bd29b73e19e6afc15def1ac999383d3642c",
+              }
+/accounts/{account}:
+  uriParameters:
+    account:
+      type: Address
+      description: Your ethereum address
+  get:
+    description: Provide information required for making solidity function request for specific address. Right now it has only nonce
+    responses:
+      200:
+        body:
+          example: |
+            {
+              "nonce": 19
+            }
+  /balances:
+    get:
+      description: Return balance for provided tokens.
+      queryParameters:
+        tokenAddresses:
+          type: array
+          items: Address
+        tokenSymbols:
+          type: array
+          items: string
+      responses:
+        200:
+          body:
+            example: |
+              [
+                {
+                  "name": "Ethereum",
+                  "symbol": "ETH",
+                  "addr": "0x0000000000000000000000000000000000000000",
+                  "listed": true,
+                  "decimals": 18,
+                  "balances": {
+                    "wallet": "1489398500000000",
+                    "exchange": {
+                      "total": "295796265000000000",
+                      "reserved": "60000000000000000",
+                      "available": "235796265000000000"
+                    }
+                  }
+                }
+              ]
+  /deposit:
+    post:
+      description: deposit etherium or tokens to smart contract via API
+      body:
+        example: |
+          {
+            tx: {
+              nonce: 18,
+              gasPrice: 1000000000,
+              gasLimit: 250000
+            },
+            signature: {
+              r: '2794e15eafe6e0a115db96f6746239f878d45face4e942ea5a797e4479008d8b',
+              s: '43081cc7453b19ed773c812ff2a472ecb2bc15ec658fe05dca9c85a1b82e5a7d',
+              v: 28
+            },
+            amount: 10000,
```

Comment:
Nit: worth to highlight that this is in decimals so maybe example should really be 1000000000000000000... (10^18 for 1 eth)